utkdigitalinitiatives/TRACE
# address TRAC-882: update the update_solr_config script (090) to refle…

…ct changes in the source configuration repository.

These changes update the `git clone` and `branch` behavior for the 090_update_solr custom shell script.

### How To Test ###
1. `vagrant destroy --force`
2. `git pull --all`
3. `git checkout -b remotes/origin/TRAC-882` (or your preferred method for checking TRAC-882 out for testing)
4. `vagrant up`
5. review the output of the startup at around the 090_... mark: are there any errors?
6. `vagrant ssh` and `cd /home/vagrant/utk-fedora-fedoragsearch-solr-config` and `git branch -a` should report `institutional_repository`

@robert-patrick-waltz @pc37utn @DonRichards @cdeaneGit
